### DISCUSSION TITLE
New WebUI interface availability

### DISCUSSION THREAD

**Discussion by claudioantonio** [git-bug/git-bug#1228] [Q&A] [answered]
How useful or worth would be an update for git-bug actual release just to offer the new WebUI interface?
It would be a new git-bug release but with the same git-bug features of the actual available release and an update only for the webui.
Off course this only makes sense if it´s not near to the next git-bug release (with enhancements and new features).

**Comment by MichaelMure** [comment, marked as answer]
In normal time, making a small release to update the webui could make sense (although it's still is non-trivial process). The problem is that right now `master` is not in a releasable state because a bunch of changes happened on the data model. A new release would make the data from a previous release unreadable. https://github.com/MichaelMure/git-bug-migration/ need to be properly finished and vetted.

Additionally, https://github.com/MichaelMure/git-bug/pull/532 is coming.
